executor/aggfuncs/builder.go

```diff
@@ -135,10 +135,26 @@ func buildBitOr(aggFuncDesc *aggregation.AggFuncDesc, ordinal int) AggFunc {
 
 // buildCount builds the AggFunc implementation for function "BIT_XOR".
 func buildBitXor(aggFuncDesc *aggregation.AggFuncDesc, ordinal int) AggFunc {
+	switch aggFuncDesc.Args[0].GetType().EvalType() {
```

Comment:
Add todo to tell that we need to implement distinct one?

Reply:
@winoros 
MySQL does not support the syntax `bit_or(distinct), bit_and(distinct), bit_xor(distinct)`

Reply:
If so, remove this [comment](https://github.com/pingcap/tidb/pull/7004/files#diff-f79c13632bdc165cfe6ed19c1a910d62R124) or add it there too?